Project 1/styles.css

```
@@ -0,0 +1,20 @@
+.card{
+        max-width: 800px;
+        margin: 30px auto;
+        background-color: #ffffff;}
+
+img{
+    max-width: 100%;}
+    title {
+        font-family: 'Slabo', serif;
+
+}
+body{    
+    font-family: 'Roboto', sans-serif;   }
+p{margin-left: 10; margin-left: 10px;}
+
+h1 h2 h3 h4 h5 h6
+        {margin-left: 10; margin-left: 10px;
```

Same note as before about this requiring units.
```suggestion
        {margin-left: 10px;
```